drivers/i2c/i2c_ll_stm32.c

```diff
@@ -511,7 +512,8 @@ void i2c_stm32_dma_rx_cb(const struct device *dma_dev, void *user_data,
 		.source_burst_length = 1,							\
 		.dest_burst_length = 1,								\
 		.dma_callback = i2c_stm32_dma_##dir##_cb,					\
-	},											\
+	}, \
+))
```

Comment:
For consistency of indentation:
```suggestion
	},))